desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -262,6 +264,53 @@ def handle_on_link_shared(host_domain, channel_id, message_ts, links, user_id):
       send_result_file(request, channel_id, message_ts, doc, 'xls')
 
 
+@login_notrequired
```

Comment:
What does that imply

Reply:
That basically the endpoint is public

Reply:
Ok, anyway from what I understand there's a verification afterwards on the user?

Reply:
A check that the user exist in Hue which will be passed by the criteo unfurler which won't be authenticated yes.